sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py

```diff
@@ -60,6 +57,8 @@ class EventGridPublisherClient():
     :param credential: The credential object used for authentication which implements
      SAS key authentication or SAS token authentication.
     :type credential: ~azure.core.credentials.AzureKeyCredential or ~azure.core.credentials.AzureSasCredential
+    :rtype: None
+    :raises: :class:`ValueError`, when events do not follow specified SendType.
```

Comment:
is this part miss placed? looks like it's related to the `send` methods

Reply:
yes - fixing it